README.md

```diff
@@ -19,6 +19,24 @@ This package provides Kolibri users with the ability to authenticate against an
 3. Restart Kolibri
 
 
+## Used claims
+This plugin will create a new user in the Kolibri database after it authenticates using the OIDC provider.
+From the [standard OIDC claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) the plugin will fetch these ones to add the information to the database:
+
+- `nickname` (or `username`)
+- `given_name`
+- `family_name`
```

Comment:
concatenated into 'full name'

Reply:
For all these comments: those are NOT kolibri fields, but the field names provided by the standard that are used to build the user info and inserted into kolibri FacilityUser. We are not free to change them, providers will use them and kolibri must handle them accordingly.